collections/_article/godotcon-us-2025-wrapup.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Looking back at GodotCon US 2025"
+excerpt: "It's time for a summary of GodotCon Boston now that all talks are available on Youtube."
+categories: ["events"]
+author: Chad Stewart
+image: /storage/app/uploads/public/56b/dd1/100/56bdd11001234567891234.jpg
+date: 2025-09-30 00:00:00
+---
+
+As you might know, this year we hosted our first US-based GodotCon in Boston. Taking place at the tail end of spring, this was three solid days focused solely on making games with the Godot Engine. Our first day had hands-on workshops followed by two days of talks. For our inaugural event we had:
+
+* Over 300 attendees.
+* Over 100 proposals.
+* 38 talks.
+* 14 showcases.
+* 3 great days.
+
+GodotCon was run by members of the Boston Godot User Group and Boston Game Dev in co-ordination with the Godot Foundation during [Boston Gamedev Week](https://gamedev.boston/). From May 5th to May 7th we had three days of Godot-focused content at the Microsoft New England Research and Development Center (NERD Center!). A huge thanks to Microsoft for generously donating the space. This would not have been possible without them and their commitment to supporting non-profits and open-source software.
+
+![Emi giving the opening keynote at the Microsoft NERD Center.](/storage/app/uploads/public/56b/dd1/100/56bdd110020250506091817.webp)
+
+And while we're at it, a huge thanks to all of our sponsors: [Arm Technology](https://www.arm.com/), [Endless Foundation](https://www.endlessglobal.com/the-endless-foundation), [W4 Games](https://www.w4games.com/), [Mike Klubnika](https://mikeklubnika.com/), [Xogot](https://xogot.com/), [Binary Solo](https://www.binarysolo.com), and [Disla Media](https://dislamedia.com/). Without them a conference like this would not be possible.
```

Comment:
```suggestion
And while we're at it, a huge thanks to all of our sponsors: [Arm Technology](https://www.arm.com/), [Endless Foundation](https://www.endlessglobal.com/the-endless-foundation), [W4 Games](https://www.w4games.com/), [Mike Klubnika](https://mikeklubnika.com/), [Xogot](https://xogot.com/), [Binary Solo](https://www.binarysolo.com/), and [Disla Media](https://dislamedia.com/). Without them a conference like this would not be possible.
```
Just to be picky :)